ui/src/main/webapp/app/table/table-component.js

```diff
@@ -308,6 +309,24 @@ function TableController(schema, results, table, events, common, types, time, cs
         return '"' + vm.sortType + '"';
     }
 
+    vm.shouldShowTruncation = function(value) {
+        return value && typeof value === 'string' && value.length > maxValueLength;
+    }
+
+    vm.getTruncatedValue = function(value) {
+        if(vm.shouldShowTruncation(value)) {
+            return value.substring(0,maxValueLength-3);
```

Comment:
You may want to add an ellipsis to show the value is truncated

Reply:
The ellipsis are added in the table.html file.

Reply:
Oh yeah. Sorry I missed that